app/forms/Mapping.scala

```diff
@@ -42,4 +42,5 @@ object Mapping {
 }
 case object NoneOfTheAbove {
   val value: String = "no"
+  val option: Option[String] = Some(value)
```

Comment:
In my opinion this is unnecessary clutter. I'd rather vote for writing `Some(NoneOfTheAbove.value)`.
The length is similar but it saves you from adding fields just because some clients need it in a different format.